packages/internet-header/src/components/post-main-navigation/post-main-navigation.tsx

```diff
@@ -286,7 +286,7 @@ export class PostMainNavigation implements HasDropdown, IsFocusable {
                           onClick={() => this.closeFlyout(levelOne.id)}
                         >
                           <span class="visually-hidden">
-                            {headerConfig.translations.mobileNavToggleClose}
+                            {levelOne.text} - {headerConfig.translations.mobileNavToggleClose}
```

Comment:
Always use a comma instead of a dash to seperate texts semantically. Because screen reader make a little break after a comma, but not after a dash.
```suggestion
                            {levelOne.text}, {headerConfig.translations.mobileNavToggleClose}
```